Madness/Map.swift

```diff
@@ -47,5 +52,12 @@ infix operator <^> {
 	precedence 130
 }
 
+/// Replace operator.
+infix operator <^ {
```

Comment:
Well, you’d still have the value on the left and the parser on the right, I think, and `value ^> parser` is really just `parser`.